src/Asserts.php

```diff
@@ -72,4 +72,18 @@ public function assertIsNotAssocArray(array $array, string $message = '')
     {
         $this->assertFalse(\array_keys($array) !== \range(0, \count($array) - 1), $message);
     }
+
+    public function assertFileIsExecutable(string $filePath, string $message = '')
+    {
+        \clearstatcache();
+
+        $this->assertTrue(\is_executable($filePath), $message);
```

Comment:
The `is_executable` should check the file is existed and executable at the same time.

You can refer the following picture:
![image](https://user-images.githubusercontent.com/9021747/55487161-2343b600-5660-11e9-8c8b-7de626e2d785.png)